src/test/java/org/apache/commons/codec/StringEncoderAbstractTest.java

```diff
@@ -72,16 +67,10 @@ public void testEncodeNull() throws EncoderException {
 
     @Test
     public void testEncodeWithInvalidObject() throws Exception {
-        boolean exceptionThrown = false;
-        try {
-            final StringEncoder encoder = this.getStringEncoder();
-            encoder.encode(Float.valueOf(3.4f));
-        } catch (final Exception e) {
-            exceptionThrown = true;
-        }
-        Assert.assertTrue("An exception was not thrown when we tried to encode " + "a Float object", exceptionThrown);
+        final StringEncoder encoder = this.getStringEncoder();
+        assertThrows(EncoderException.class, () -> { encoder.encode(Float.valueOf(3.4f)); },
```

Comment:
No curlies needed; I'll assume you'll go over the other files. I'll review again after you push.

Reply:
I tried to not change or remove any of the messages (except for typos), even if I think the message was not necessary. All I did was to change the position of the message as a parameter, since this changed from JUnit 4 to 5. My intention was to minimize the changes, and afterall, some of the messages give information about the failure of the test that the message from JUnit will not give.

Reply:
My comment was not about assertion messages, but about the extra {} that are superfluous.

Reply:
Yes, sorry not letting this clear. I understood and will remove them. I only commented about the messages because when I started the rebase process (as instructed by @aherbert), I noticed that some messages were removed.